src/SDKs/DataMigration/Management.DataMigration/Properties/AssemblyInfo.cs

```diff
@@ -7,8 +7,8 @@
 [assembly: AssemblyTitle("Microsoft Azure Data Migration Management Library")]
 [assembly: AssemblyDescription("Provides management functionality for Microsoft Azure Data Migration Resources.")]
 
-[assembly: AssemblyVersion("0.1.2")]
-[assembly: AssemblyFileVersion("0.1.2")]
+[assembly: AssemblyVersion("0.2.0")]
```

Comment:
Again, you don't need to update the assembly version here unless its a major version change.

Reply:
Ah it was merged when I did git pull from Azure/azure-sdk-for-net. Sorry for confusion.